module/actor/mortal-actor-sheet.js

```diff
@@ -188,9 +188,12 @@ export class MortalActorSheet extends CoterieActorSheet {
             );
           const modifier = parseInt(html.find("#inputMod")[0].value || 0);
           const difficulty = parseInt(html.find("#inputDif")[0].value || 0);
+          const useWound = parseInt(html.find("#useWound")[0]?.checked || false);
```

Comment:
This should not parseInt. This will always return NaN.

Reply:
Also, don't feel the need to fix this now because of me. I actually use my own custom version of this sheet until this one gets some love from its owner . I'm just trying to be nice and help iron out the kinks until the owner gets back ^^